lib/BloqadeSchema/src/types.jl

```diff
@@ -239,27 +239,27 @@ get_device_capabilities() = DeviceCapabilities(
 # leave as SI units, needed for rounding purposes
 get_device_capabilities_SI() = DeviceCapabilities(
     task=TaskCapabilities(
-        numberQubitsMax = 100,
+        numberQubitsMax = 256,
```

Comment:
- This is moved into the 
`Lattice` type.

Reply:
As in, it's dropped in favor of using `numberSitesMax` which falls in `LatticeGeometryCapabilities` under `LatticeCapabilities`?